### PR TITLE
DOP-3953

### PR DESCRIPTION
**Pull request by seungpark**
### Stories/Links:

DOP-3953
This PR removes all references of `GATSBY_TEST_SEARCH_UI` within the front end. Behavior should still be the same as prod as we have flipped the switch. (minus pagination, that hasn't been released into prod yet, but is not part of changes within this branch)

### Current Behavior:

[on prod](https://www.mongodb.com/docs/search/?q=test&page=1&searchProperty=manual-v6.3&searchVersion=6.3+%28current%29)

### Staging Links:

[staged changes](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/landing/seung.park/DOP-3953/search/index.html?q=test&page=1&searchProperty=manual-v6.3)

### CODE REVIEW

File: src/components/SearchResults/EmptyResults.js [hunk 1]
```diff
@@ -78,55 +33,18 @@ const SupportingText2 = styled('p')`
...
-            </SupportingText2>
-          </NoResultText>
-        </EmptyStateContainer2>
-      )}
-    </>
+    <EmptyStateContainer>
+      <NoResults />
+      <NoResultText>
+        <H3>No results found</H3>
+        <SupportingText2>
```
Comment (mmeigs) [inline src/components/SearchResults/EmptyResults.js:42]: nit: Could we rename this without the `2`?

**Review by mmeigs — APPROVED** [review]
Looks good. One comment. Github made that diff real confusing lol

File: src/components/SearchResults/SearchResult.js [hunk 2]
```diff
@@ -5,15 +5,12 @@ import styled from '@emotion/styled';
 import { palette } from '@leafygreen-ui/palette';
 import { Body } from '@leafygreen-ui/typography';
 import { theme } from '../../theme/docsTheme';
-import Tag, { searchTagStyle, searchTagStyleFeature } from '../Tag';
+import Tag, { searchTagStyleFeature } from '../Tag';
```
Comment (rayangler) [inline src/components/SearchResults/SearchResult.js:8]: (Nit) We can probably consolidate `searchTagStyleFeature` with `searchTagStyle` now

**Review by rayangler — COMMENTED** [review]
Pretty much lgtm, just one non-blocking comment, take it or leave it

File: src/components/SearchResults/EmptyResults.js (hunk 1 above)
Comment (seungpark) [reply src/components/SearchResults/EmptyResults.js:42]: good catch! done

File: src/components/SearchResults/SearchResult.js (hunk 2 above)
Comment (seungpark) [reply src/components/SearchResults/SearchResult.js:8]: good call! done

**Review by rayangler — APPROVED** [review]

**Pull request merged**